packages/lib/src/components/UIElement.tsx

```diff
@@ -78,8 +78,7 @@ export class UIElement<P extends UIElementProps = any> extends BaseElement<P> im
 
         const { type } = analyticsObj;
 
-        // let storedCardIndicator;
-        if (type === ANALYTICS_RENDERED_STR) {
+        if (type === ANALYTICS_RENDERED_STR || type === ANALYTICS_CONFIGURED_STR) {
             // Check if it's a storedCard
```

Comment:
Would it be possible to let the corresponding children components (e.g. `scheme` and `bcmc`) send the correct data `component` and `analyticsObj`?

I think the UIElement doesn't need to know anything about the children component, whether it's a stored card or not.

Reply:
I agree this is Card specific logic... I'll make the change